src/ui/base-page-storyEdit.js

```diff
@@ -222,25 +227,45 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
         return filteredBlocks;
     };
 
-    /* Returns true if a block is determined to be empty, based on the values
+    /**
+     * Returns true if a block is determined to be empty, based on the values
      * listed in blockContentValuesForType. If at least one of those values is
      * truthy, the block is not empty. If the values are empty or otherwise can't
      * be iterated over, then the block is also empty regardless of its contents.
-     * - "block": a single story block (sjrk.storyTelling.block)
-     * - "blockContentValuesForType": an array of values for the block's type
+     *
+     * @param {Object} block - a single story block (sjrk.storyTelling.block)
+     * @param {Object} blockContentValuesForType - an array of values for the block's type
      *      that, if at least one is truthy, mean the block is not empty
+     *
+     * @return {Boolean} - true if the block is considered empty
      */
     sjrk.storyTelling.base.page.storyEdit.isEmptyBlock = function (block, blockContentValuesForType) {
         return !fluid.find_if(blockContentValuesForType, function (blockContentValue) {
             return !!block[blockContentValue];
         });
     };
 
+    /**
+     * Depending on whether story authoring is enabled, will set/add CSS classes
+     * and HTML properties to show or hide parts of the page
+     *
+     * @param {Object} mainContainer - the main page container (mid-level)
```

Comment:
What does "mid-level" and "top-level" mean?

Reply:
I've removed any logic relating to mainContainer, so the distinction between the elements' hierarchy is no longer relevant